src/playground/utils/codemirror-linter-extension.js

```diff
@@ -92,11 +92,11 @@ function setDiagnostics(state, diagnostics) {
  * The state effect that updates the set of active diagnostics. Can
  * be useful when writing an extension that needs to track these.
  */
-const setDiagnosticsEffect = /* @__PURE__*/StateEffect.define();
-const togglePanel = /* @__PURE__*/StateEffect.define();
-const movePanelSelection = /* @__PURE__*/StateEffect.define();
+const setDiagnosticsEffect = /** @__PURE__*/StateEffect.define();
```

Comment:
Just double-checking -- this doesn't appear to be a JSDoc comment, so maybe the extra asterisk shouldn't be there?

Reply:
Correct, fixed.